src/components/MapsSection/MapsSection.tsx

```diff
@@ -100,9 +107,20 @@ const MapsSection = ({
       <SectionHeader id="maps-visu" sectionHead={sectionHead} />
       <BoxWrapper>
         <PreviewWrapper>
-          <ExpandBox href={`/map?name=${currentVisu.id}`}>
-            <Icon id="expand" size={18} />
-          </ExpandBox>
+          <ButtonsWrapper>
+            <ExpandBox
+              href={`/map?name=${currentVisu.id}`}
+              title={`Expandir mapa: ${currentVisu.name}`}
+            >
+              <Icon id="expand" size={18} />
+            </ExpandBox>
+            <PinBox
+              onClick={handlePin}
+              title={`Fixar mapa: ${currentVisu.name}`}
+            >
+              <Icon id={`${pinMap ? "un" : ""}pin`} size={18} />
```

Comment:
Just a comment here, I think would be better to implement this like this:

```suggestion
              <Icon id={pinMap ? "unpin" : "pin"} size={18} />
```

I know that they have the same subword "pin" and you're trying to improve a kinda coding duplication, but sometimes this makes the code more difficult to read and understand - When I was reading I cought myself thinking "the id of this icon is 'un'?".

Reply:
Yeah, makes a lot of sense!! Fixed that. Thanks!!

Reply:
also, this made me fix some wrong title in the pin button. I added the change in the title, now when hovering in the button if pinned the title is "Desafixar mapa: {mapa}", otherwise is "Fixar mapa: {mapa}"

Reply:
That's great!